### DISCUSSION TITLE
GEO ip blocking

### DISCUSSION THREAD

**Discussion by killmasta93** [DigitalRuby/IPBan#188] [Ideas]
HI i was wondering if in the future possible to add geo ip blocking?

**Comment by jjxtra** [comment]
Not likely because the windows firewall would fall over on itself. For ipban pro I use the lower level windows filtering platform which performs way better and therefore ipban pro does support blocking countries. You can also invert the country block so that all other packets not in your specified countries are blocked.

**Reply by killmasta93** [reply]
Thank you for the reply ill keep that in mind

**Reply by maxspyone** [reply]
I am also interested in Geo blocking or Geo IP whitelist. What is the difference to the pro version? Why is the Windows Firewall used with the free?

**Reply by jjxtra** [reply]
The pro version allows country blocking. The free version does not. The free version only supports Windows firewall which will fall over on itself if you try to block countries. The pro version uses Windows Filtering Platform which has no problem blocking hundreds of thousands of ip addresses / cidr ranges.
